src/Components/Http/Installer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of Laravel Zero.
+ *
+ * (c) Nuno Maduro <enunomaduro@gmail.com>
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ */
+
+namespace LaravelZero\Framework\Components\Http;
+
+use LaravelZero\Framework\Components\AbstractInstaller;
+
+/**
+ * @internal
+ */
+final class Installer extends AbstractInstaller
```

Comment:
There is no service provider?

Reply:
It didn't seem to be necessary to use the client. 🤔

Reply:
I've tested quite a lot of the functionality provided by the `Http` client, and it seems to work without requiring a provider. 👍

I've added a new test that checks that it functions correctly. 👍